### PR TITLE
Mobile Release v1.112.0

### PR DESCRIPTION
**Pull request by derekblank**
## Description
Release 1.112.0 of the react-native-editor and Gutenberg-Mobile.

- https://github.com/wordpress-mobile/gutenberg-mobile/pull/6597
- https://github.com/wordpress-mobile/WordPress-Android/pull/20093
- https://github.com/wordpress-mobile/WordPress-iOS/pull/22513

### CODE REVIEW

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

File: packages/react-native-editor/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+
+## 1.112.0
```
Comment (fluiddot) [inline packages/react-native-editor/CHANGELOG.md:14]: @derekblank I noticed when checking the release notes in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6597 that several entries are missed in this section:
- https://github.com/WordPress/gutenberg/pull/58266
- https://github.com/WordPress/gutenberg/pull/57869 ([reference](https://github.com/WordPress/gutenberg/commit/7b77931ec359523f7baadd9abd15f685de2a7f04#r138159516))

**Review by fluiddot — CHANGES_REQUESTED** [review]

File: packages/react-native-editor/CHANGELOG.md (hunk 1 above)
Comment (fluiddot) [reply packages/react-native-editor/CHANGELOG.md:14]: I took the liberty to apply this suggestion in https://github.com/WordPress/gutenberg/pull/58540/commits/8554e25e1bb4a3b17fe090d4ad7ae073b67ea05b.

**Review by fluiddot — APPROVED** [review]

File: packages/react-native-editor/CHANGELOG.md (hunk 1 above)
Comment (derekblank) [reply packages/react-native-editor/CHANGELOG.md:14]: Thank you for updating -- these CHANGELOG updates were generated by the release CLI, and indeed the image-related entry was listed under the incorrect release. I'll check the release CLI if there was an issue, or if it was due to listing being in the incorrect release.

**Pull request merged**